cttportfolio.css

```diff
@@ -0,0 +1,982 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+	color: rgb(4, 8, 9);
+	border-radius: 7px;
+}
+
+body {
+	margin: 0;
+	background-color: rgb(2, 8, 9);
+}
+
+main>div {
+	margin: 0 auto;
+	max-width: 1300px;
+}
+
+main {
+	margin-top: 200px;
+}
+
+h4,
+h5,
+h6,
+.pp_body h1 {
+	font-weight: 700;
+	font-size: 6.0rem;
+	text-transform: uppercase;
+	-webkit-text-stroke-width: 0.3px;
+	-webkit-text-stroke-color: black;
+	color: rgb(248, 248, 255);
+}
+
+h5 {
+	font-weight: 600;
+	font-size: 4rem;
+}
+
+.am_text,
+button,
+.navlinks a,
+.contactmetext>div,
+img {
+	box-shadow: 0px -2px 5px rgba(2, 8, 9, 0.6)
+}
+
+button {
+	color: white;
+	background: none;
+	padding: 10px;
+	font-size: 2rem;
+	text-transform: uppercase;
+	border: 1px solid rgb(248, 248, 255);
+	font-weight: 600;
+	border-radius: 5px;
+}
+
+a.icon_hover img,
+a.pf_sub_links h6,
+.navlinks.icon_hover>a:hover {
+	border: none;
+}
+
+a.icon_hover:hover img,
+a.pf_sub_links:hover h6,
+button:hover,
+.navlinks.icon_hover>a:hover {
+	border: 1px solid rgba(248, 248, 255, 1);
+	box-shadow: 0px -2px 5px rgba(248, 248, 255, 0.6);
+}
+
+.background_color,
+.project_text>* {
+	background-color: rgba(255, 182, 193, 0.7);
+	box-shadow: 0px -2px 5px rgba(2, 8, 9, 0.6);
+	border-radius: 5px;
+}
+
+h1 {
+	animation: 10s fadeIn1;
+	animation-fill-mode: forwards;
+	visibility: hidden;
+}
+
+h2,
+.pp_info>div {
+	animation: 10s fadeIn2;
+	animation-fill-mode: forwards;
+	visibility: hidden;
+}
+
+nav {
+	animation: 10s fadeIn3;
+	animation-fill-mode: forwards;
+	visibility: hidden;
+}
+
+@keyframes fadeIn1 {
+	10% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	25% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+@keyframes fadeIn2 {
+	25% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	40% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+@keyframes fadeIn3 {
+	40% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	65% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+
+/*{ Keeping this stored here for now for access later if needed with browser testing
```

Comment:
it's good practice to only store what is used, this commented code needs to be removed. Code can 'decay' over time and this code might not work as the surrounding changes.